custom-code-library.hbs

```diff
@@ -64,7 +64,7 @@
             <div>
                 {{#get "tags" limit="all"}}
                     {{#foreach tags visibility="internal"}}
-                        <h1>{{'name'}} Routines</h1>
+                        <h1>{{{trimString name}}} Routines</h1>
```

Comment:
@orklepee where does `name` come from? Does it need those single quotes around it? I'm thinking there's probably another `name` defined somewhere and this one is getting clobbered which would explain why it silently "fails"

Reply:
@justinsantoro `name` is an attribute of Ghost tags. It does not require single quotes around it, but it does work with and/or without them. Currently, if I remove "trimString" from the brackets, the internal tags are visible and their associated articles appear in dev.nolongerset/codelibrary/.

Reply:
ok thanks.